src/components/ToDoComponent/TodoComponent.tsx

```diff
@@ -3,42 +3,54 @@ import React, { useEffect, useState } from 'react';
 import Card from './Card';
 import data from './Data';
 import { TodoStyles } from './Styles/TodoStyles';
+import Task from './taskType';
 
 const TodoComponent = () => {
   const [tasks, setTasks] = useState(data);
```

Comment:
const [tasks, setTasks] = useState<Task[]>(data);

Reply:
made this change in recent commit